### DISCUSSION TITLE
💻 Copilot: An interesting keyboard shortcut...

### DISCUSSION THREAD

**Discussion by jacob-leger** [community/community#17841] [Other Feature Feedback, Questions, & Ideas] [answered]
## The problem, if it is a problem.

When I first installed the extension in the VSCode IDE, I noticed that
one of my most used, if not **the** most used keyboard shortcut was
used by Copilot.
That is the Copilot Extension's opening shortcut: "Ctrl+Enter".

"Ctrl+Enter" is very useful, it creates a new line, wherever you are
in the line of text, you are in. 
Before I used to know this shortcut,
I used to click the end of the current line I was on, and then
press enter. 
This is very inefficient, you should minimize the use of the mouse while coding. I know this might seem picky, but I think there might be
a better solution.

## The solution, but it's up to you.

I know that you can change keyboard shortcuts in VSCode, but I think
that it could be changed to something like "Ctrl+Alt+C", or anything
the developers think is fit.

Most devs use VSCode, so I think the Copilot Engineers can come up with
something, maybe even an unused shortcut if they want.

Here are some polls deciding the most used IDE:
 - https://www.youtube.com/post/Ugkxbr1P4le65COnBQ5iERcs6ejZCQvprhKU

Please vote and comment on what you think down below, anything counts.

**Comment by sanskaromar** [comment, marked as answer]
I would like to give an alternative to using the mouse to go to the end of the line and then press ENTER, instead you can press **END** to go to the end of the line and then press ENTER.

Edit: For similar shortcuts like this, 
* you can press **HOME** to go to the start of the line
* [VS Code shortcuts pdf for windows](https://code.visualstudio.com/shortcuts/keyboard-shortcuts-windows.pdf)
* [VS Code shortcuts for linux](https://code.visualstudio.com/shortcuts/keyboard-shortcuts-linux.pdf)
* [VS Code shortcuts for macOS](https://code.visualstudio.com/shortcuts/keyboard-shortcuts-macos.pdf)

**Reply by jacob-leger** [reply]
Thank you, this will help a lot.

**Reply by jacob-leger** [reply]
Do you know any other shortcuts like this?

**Reply by sanskaromar** [reply]
Yep, I have updated them in my original answer.

**Reply by jacob-leger** [reply]
Thx

**Comment by ketphan02** [comment]
I also find that the copilot "tab" shortcut is a bit annoying since it stops me from indenting my code. How about command + tab 🤔

**Reply by jacob-leger** [reply]
I agree!
